build-system/tasks/performance/stats.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const average = array => {
+  if (!array || array.length == 0) {
+    return 0;
+  }
+  return array.reduce((a, b) => a + b) / array.length;
+};
+
+const sort = array => {
+  return array.sort((a, b) => {
+    return a - b;
+  });
+};
```

Comment:
Nit: This is a good place to use the terser format. (This is really a nit, no need to change).

```suggestion
const sort = array => array.sort((a, b) => a - b);
```